Virtusize/Tests/APIRequestTests.swift

```diff
@@ -157,10 +157,11 @@ class APIRequestTests: XCTestCase {
         XCTAssertNotNil(apiRequest?.httpBody)
         let actualParams = try? JSONDecoder().decode(VirtusizeGetSizeParams.self, from: apiRequest!.httpBody!)
         XCTAssertNotNil(actualParams)
-        XCTAssertEqual(actualParams?.additionalInfo.count, 5)
-        XCTAssertEqual(actualParams?.additionalInfo["gender"]?.value as? String, "male")
+        XCTAssertEqual(actualParams?.items.first?.additionalInfo.count, 4)
```

Comment:
What 4 means ? I feel it is a magic number 🤔

Reply:
in test cases this number will be dynamic to check other senerios,

Reply:
Understood thanks